src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/nomis/GetAddressesForPersonTest.kt

```diff
@@ -74,7 +72,7 @@ class GetAddressesForPersonTest(
     addresses.shouldBeEmpty()
   }
 
-  it("throws an exception when 404 Not Found is returned") {
+  it("returns null when 404 Not Found is returned") {
```

Comment:
For consistency with PersonControllerTest

```suggestion
  it("returns null when 404 NOT FOUND is returned") {
```